resources/css/style.css

```diff
@@ -45,9 +45,22 @@
 .title-get-involved {
   font-family: -apple-system, BlinkMacSystemFont, Roboto, sans-serif;
   font-size: 5em;
-  font-weight: 100;
+  font-weight: 300;
   text-align: center;
 }
+.row .description {
+  font-family: Roboto;
+  font-size: 1.6em;
+  text-align: justify;
+  font-weight: 300;    
```

Comment:
@Shruti9520 There is extra whitespace on Line 55 after ```font-weight:300```, so you'll have to remove that extra whitespace. And a new PR shouldn't be created after every review iteration, you can follow the [newcomer guide](http://api.coala.io/en/latest/Developers/Newcomers_Guide.html#step-8-review-process) to see how to _amend your commit_ and to _push the changes_ in this PR itself 😃